docker/build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.24.6-bullseye@sha256:2cdc80dc25edcb96ada1654f73092f2928045d037581fa4aa7c40d18af7dd85a
+FROM golang:1.25.3-bookworm
```

Comment:
@Kgrah The versions in the docker file MUST pin the version using the `@sha256:<digest>` form.